client/data/settings/selectors.js

```diff
@@ -184,6 +184,10 @@ export const getPaymentRequestButtonTheme = ( state ) => {
 	return getSettings( state ).payment_request_button_theme || '';
 };
 
+export const getPaymentRequestButtonBorderRadius = ( state ) => {
+	return getSettings( state )?.payment_request_button_border_radius ?? 10;
```

Comment:
When ECE is disabled that implies that the Payment Request Buttons are enabled instead. Those do not support changing the border radius, so we need a fallback value when ECE is disabled.

A default radius of 10px was something I just chose arbitrarily while working on the PR. We'll probably want some other default radius for when ECE is disabled, likely to match what the PRBs use.

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/9010/commits/b76d492b54b2137d1193677bfc225d9e5dd0658e